### PR TITLE
History page

### PR DESCRIPTION
**Pull request by partha8**
# Note
To see the functionality work follow the below steps:
1. Log in as a guest
2. Go to explore the page (2nd icon on sidebar)
3. Click on any one of the videos, play it.  _**The video will be added only after you start playing the video.**_
4. Go to the history page (last icon)

## Files to check
1. [src/components/HorizontalVideoCard/HorizontalVideoCard.js](https://github.com/partha8/weeb-culture/compare/history-page?expand=1#diff-3142f140c59a9c85c45962c1e0fa378602afd3938f723872cc14370e6c283cc4)
6. [src/components/HorizontalVideoCard/horizontal-video-card.css](https://github.com/partha8/weeb-culture/compare/history-page?expand=1#diff-05a51fbb393eb251d64e7342106458648b4aa1af0e561477f65971d9ca4cbd3a)
7. [src/hooks/useGetHistory.js](https://github.com/partha8/weeb-culture/compare/history-page?expand=1#diff-6677e817cfd00dca9a88ed82563b33b9b29c21dfded4334a911ff7bf67af011d)
8. [src/pages/History/History.js](https://github.com/partha8/weeb-culture/compare/history-page?expand=1#diff-22b44aced7ecb505ce1c48f26fad8db7c618e7e92f280ef68d82756a94d35dc8)
9. [src/utils/videoUtils.js](https://github.com/partha8/weeb-culture/compare/history-page?expand=1#diff-4fa4d6caaf3f157564aa811b70773ffdd975fbfb8233c0703d13977eb11250e8)

### CODE REVIEW

File: src/pages/History/History.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export const History = () => {
+  return <div>History</div>;
+};
```
Comment (MayankKumar10) [inline src/pages/History/History.js:35]: Remove the history CSS empty file if it's not needed

**Review by MayankKumar10 — APPROVED** [review]
Just a suggestion everything else looks good you can merge the changes.

File: src/pages/History/History.js (hunk 1 above)
Comment (partha8) [reply src/pages/History/History.js:35]: This is an old commit, the latest commits/changes have the history page functionalities and the history.css does contain styles for the same.

**Review by hsnice16 — APPROVED** [review]
UI is great, brother. The code is clean too. Nothing found to suggest for now. Good work.

**Pull request merged**